addons/website_event/models/event_event.py

```diff
@@ -97,14 +94,6 @@ def _default_question_ids(self):
     start_remaining = fields.Integer(
         'Remaining before start', compute='_compute_time_data',
         help="Remaining time before event starts (minutes)")
-    # questions
-    question_ids = fields.One2many(
```

Comment:
While moving the code, could you also move the tests ? Notably ``test_event_type_configuration_from_type``, the ``TestEventQuestionCommon`` class. Did not found other stuff, to move, but looked quickly so maybe have another look to check if we can move some other stuff linked to questions :)